validator/validator-main.protoascii

```diff
@@ -5819,6 +5819,7 @@ attr_lists: {
   attrs: { name: "[aria-valuenow]" }
   attrs: { name: "[aria-valuetext]" }
   attrs: { name: "[class]" }
+  attrs: { name: "[class+]" }
```

Comment:
@honeybadgerdontcare The pattern I was trying here locally was...

```
attrs: {
  name: "[class]"
  trigger: {
    mandatory_oneof: "['[class]', '[class+]']"
  }
}
attrs: {
  name: "[class+]"
  trigger: {
    mandatory_oneof: "['[class]', '[class+]']"
  }
}
```

Going to likely need a custom specific rule here or the ability to indicate mutually exclusive attributes like this for global elements. Another option is to error in `amp-bind` for this scenario.

Reply:
I'm inclined to something like `disallowed_attr` as a new field for attributes that would act similar to the trigger. The trigger was introduced depending on the value of the attribute. I don't think we need to expand that and could put this on the attr spec. Reviewing with wg-caching.